functional_tests/test_db_online_offline.py

```diff
@@ -142,10 +144,37 @@ def rest_scan(sync_gateway, db, online, num_docs, user_name, channels):
             log.error((e.response.url, e.response.status_code))
             error_responses.append((e.response.url, e.response.status_code))
 
+    # PUT /{db}/{local-doc-id}
+    local_doc_id = uuid.uuid4()
+    try:
+        doc = user.add_doc("_local/{}".format(local_doc_id), content={"message": "I should not be replicated"})
+    except HTTPError as e:
+        log.error((e.response.url, e.response.status_code))
+        error_responses.append((e.response.url, e.response.status_code))
+
+    # GET /{db}/{local-doc-id}
```

Comment:
Do you mean the following?

A scenario where user-1 puts _local/doc_1. Then user-2 puts _local/doc_1 to the same sync_gateway. Assert that there is a conflict error for user-2?